### PR TITLE
Refactoring preferences

### PR DESCRIPTION
**Pull request by nittka**
This PR primarily addresses #186, but it includes the changes of PR #185.

### CODE REVIEW

File: org.elysium.parent/org.elysium.ui/src/org/elysium/ui/preferences/LilyPondRefactoringPreferencePage.java [hunk 1]
```diff
@@ -1,20 +1,61 @@
...
+				setEnablement(dependentFields, getBooleanValue());
+			}
+		};
+		addField(enabledEditor);
+		dependentFields.add(
+				createBooleanField(REFACTORING_DELETE_COMPILED, "Delete compiled (pdf, midi) when deleting source"));
+		dependentFields.add(
+				createBooleanField(REFACTORING_ADAPT_INCLUDES, "Modify includes pointing to refactored files"));
+		dependentFields.add(
+				createBooleanField(REFACTORING_WARN_DELETE_INCLUDED, "Warn if an indluded file is deleted"));
```
Comment (thSoft) [inline org.elysium.parent/org.elysium.ui/src/org/elysium/ui/preferences/LilyPondRefactoringPreferencePage.java]: Please fix typo: indluded -> included

File: org.elysium.parent/org.elysium.ui/src/org/elysium/ui/preferences/LilyPondRefactoringPreferencePage.java [hunk 2]
```diff
@@ -1,20 +1,61 @@
...
+		dependentFields.add(
+				createBooleanField(REFACTORING_WARN_SEARCHPATH, "Warn if a search path is affected by a refactoring"));
+		dependentFields.add(
+				createBooleanField(REFACTORING_WARN_VARIABLE_INCLUDE, "Warn if a there are variable includes (refactoring not supported)"));
```
Comment (thSoft) [inline org.elysium.parent/org.elysium.ui/src/org/elysium/ui/preferences/LilyPondRefactoringPreferencePage.java]: Please fix typo: if a -> if

**Comment by nittka** [comment]
I fixed the typos and rebased on master.
I assume you still would not like to have the preference for switching off the include modifications. If you changed your mind, adding the additional preference would be easy.

**Comment by thSoft** [comment]
Great, thanks!

**Pull request merged**